pkg/web/api/v1/api.go

```diff
@@ -26,10 +27,21 @@ type Client interface {
 	UpdateTransaction(context.Context, *Transaction) (*Transaction, error)
 	DeleteTransaction(context.Context, uuid.UUID) error
 
+	// Transaction Actions
+	Prepare(context.Context, *Prepare) (*Prepared, error)
+	SendPrepared(context.Context, *Prepared) (*Transaction, error)
+	Export(context.Context, io.Writer) error
+
+	// Transaction Detail Actions
+	Preview(ctx context.Context, transactionID uuid.UUID) (*Envelope, error)
+	SendEnvelope(ctx context.Context, transactionID uuid.UUID, in *Envelope) (*Envelope, error)
+	Accept(ctx context.Context, transactionID uuid.UUID, in *Envelope) (*Envelope, error)
```

Comment:
These two method signatures are identical (and suffer from the same problem as the TransactionDetail, TransactionInfo, and AcceptTransactionPreview). 

@daniellemaxwell would it be possible that `Accept` accepts the latest version of the envelope, but if there are edits the form uses the `Send` endpoint? I guess it does depend on the semantics, because `accept` seems to indicate that the case is closed. While send can be an intermediate.